docs/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
Is there a requirement to license the docs this way?
I think the correct approach would be not to have a separate LICENSE file here, but include copyright notices that refer to the top-level licensing notices.
Thoughts @ngmr @jakub-pomykala ?

Reply:
So we are ok to license our documentation with the same license as the rest of Yoko? That would be simplest, and should need just one LICENSE file and possibly one NOTICES file alongside our README